docs_src/src/installation.md

```diff
@@ -57,6 +64,29 @@ Install dependencies for `circom-helper` and `zkey-manager`:
 sudo apt-get install libgmp-dev nlohmann-json3-dev nasm g++
 ```
 
+### Configure circom-helper and zkey-manager
+
+Edit `circuits/circomHelperConfig.json` to include the relative path to the
+circom binary.
+
+```json
+    "circom": "RELATIVE_PATH_TO_CIRCOM",
+    "snarkjs": "./node_modules/snarkjs/build/cli.cjs",
+    "circuitDirs": [
+        "./circom/test/"
+    ]
+}
+```
+
+Edit `cli/zkeys.config.yml` to include the relative path to the
```

Comment:
Are we no longer pulling this file from the Azure instance?

Reply:
Just uploaded the zkey files to a new Azure container, and pushed a new commit with an updated download script.